prow/report/report.go

```diff
@@ -78,8 +94,8 @@ func reportStatus(ghc GithubClient, pj v1.ProwJob, childDescription string) erro
 		}
 		if err := ghc.CreateStatus(refs.Org, refs.Repo, refs.Pulls[0].SHA, github.Status{
```

Comment:
There are other calls to `ghc.CreateStatus`, do we want to move truncate inside of github client?

Reply:
ok - we can wait until someone else also need to truncate :-)